theme/themes/pastanaga/extras/main.less

```diff
@@ -35,6 +35,10 @@ body.cms-ui .ui.basic.segment.content-area {
   cursor: text;
 }
 
+#page-login > .container {
+  margin: auto !important;
```

Comment:
I'm not comfortable with the `!important` here. Could you point to the setting that it's trying to "fix"? Why is the container working in the other pages?

Reply:
because these stylings are getting inherited from semantic.less (which is not expected)
![image](https://user-images.githubusercontent.com/67837886/229489734-e3c8fef8-c961-4cfe-8a67-8331aac037f7.png)

So i have done the override of styling for this specific login page container by specifying in ```#page-login > .container``` selector with !important